src/routers/course.ts

```diff
@@ -1,24 +1,65 @@
 import express from 'express';
+import { create, read, update, del } from '../operators/courseOperations';
 
 const courseRouter = express.Router();
+const app = express();
+
+app.use(express.json());
+
+courseRouter.get('/:id', async (req, res) => {
+	try {
+		const id = req.params.id;
+		const userID = req.header('userID');
+
+		if (id && userID) {
+			const course = await read(id, userID);
+			res.send(course);
+		} else {
+			throw 'ERROR - id undefined';
+		}
+	} catch (err) {
+		res.status(404).send(err);
```

Comment:
We want to make sure that we just return a generic `Not Found` here so the user doesn't know the error (error message really only help hackers, not real users since they shouldn't encounter this issue). `res.status(404).send("Not Found");`